test/execution/test_placeholder.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import torch as th
+
+from syft.execution.placeholder import PlaceHolder
+
+
+def test_placeholder_forwarding():
```

Comment:
I'm probably missing something, but it doesn't seem like this test requires the other Crypten-related changes in this PR?

Reply:
Currently, I want to make sure I got the coverage rate > 95% (and see what else it fails on the tests). After that, I want to rebase the CrypTen branch with what it is on the master (and those changes should disappear).